docs/book/quick-start.md

```diff
@@ -1,211 +1,95 @@
-pulling variables and the template name from it. By default, you can omit the
-template name, and it will resolve to "lowercase-module-name/lowercase-controller-name/lowercase-action-name".
-However, you can override this to specify something different by calling
-`setTemplate()` on the `ViewModel` instance. Typically, templates will resolve
-to files with a ".phtml" suffix in your module's `view` directory.
-
-So, with that in mind, let's create a view script.
+By default, this controller will render the view script located in `view/application/hello/world.phtml`.
+You can customize this behavior.
+Learn more about views [here](https://docs.laminas.dev/laminas-view/quick-start/).
 
 ## Create a View Script
 
-Create the directory `view/<module-name>/hello`. Inside that directory, create a
-file named `world.phtml`. Inside that, paste in the following:
+Create the file `view/application/hello/world.phtml` and add the following code:
 
 ```php
 <h1>Greetings!</h1>
 
 <p>You said "<?php echo $this->escapeHtml($message) ?>".</p>
 ```
 
-That's it. Save the file.
-
 > ### Escaping output
 >
-> What is the method `escapeHtml()`? It's actually a [view helper](https://docs.laminas.dev/laminas-view/helpers/intro/),
-> and it's designed to help mitigate XSS attacks. Never trust user input; if you
-> are at all uncertain about the source of a given variable in your view script,
-> escape it using one of the provided escape view helpers depending on the type
-> of data you have.
-
-## View scripts for module names with subnamespaces
-
-As per PSR-0, modules should be named following the rule: `<Vendor Name>\<Namespace>\*`.
-
-Since version 3.0, the default template name resolver uses fully qualified
-controller class names, stripping only the `\Controller\\` subnamespace, if
-present.  For example, `AwesomeMe\MyModule\Controller\HelloWorldController`
-resolves to the template name `awesome-me/my-module/hello-world` via the
-following configuration:
-
-```php
-'view_manager' => [
-    'controller_map' => [
-        'AwesomeMe\MyModule' => true,
-    ],
-],
-```
-
-(In v2 releases, the default was to strip subnamespaces, but optional mapping rules
-allowed whitelisting namespaces in module configuration to enable current
-resolver behavior. See the [migration guide](migration/to-v3-0.md#laminasmvcviewinjecttemplatelistener)
-for more details.)
+> The method `escapeHtml()` is a [view helper](https://docs.laminas.dev/laminas-view/helpers/intro/), and it's designed to help mitigate XSS attacks.
+> Never trust user input.
+> If you are at all uncertain about the source of a variable in your view, escape it using one of the view helpers, depending on the type of data.
 
 ## Create a Route
 
 Routes determine which controller to call based on the URI and other information from the request.
 
-Configure a route and a controller:
+Configure a route and a controller in `config/module.config.php`:
```

Comment:
The controller is added to the `Application` module, so the configuration of this module should be used. Otherwise we must explain why it is configured _outside_.

At the moment, there is also no information that the application module will be extended in this quick start.

Reply:
@Xerkus You're right, this is the correct path according to: https://github.com/laminas/laminas-mvc-skeleton/blob/2.0.x/module/Application/config/module.config.php

@froschdesign
I don't understand what you mean by this:

> there is also no information that the application module will be extended in this quick start

Reply:
@afilina 
> I don't understand what you mean…

In the quick start, the information is missing that the module `Application` is modified or generally that it is a module.